include/frozen/map.h

```diff
@@ -236,6 +241,12 @@ class map<Key, Value, 0, Compare> {
   constexpr key_compare key_comp() const { return compare_; }
   constexpr key_compare value_comp() const { return compare_; }
 };
+
+template <typename T, typename U, std::size_t N>
+constexpr auto make_map(std::tuple<T, U> const (&items)[N]) {
+  return map<T, U, N>{bits::to_array(items)};
+}
```

Comment:
That's a style comment, but I'd rather have this written:

```c++
constexpr map<T, U, N> make_map(std::tuple<T, U> const (&items)[N]) {
    return {bits::to_array(items)};
```

Reply:
When I write them all that way, I get a compiler error with clang-4.0

    ../include/frozen/set.h:195:10: error: chosen constructor is explicit in copy-initialization
      return {bits::to_array(args)};
             ^~~~~~~~~~~~~~~~~~~~~~

I get a similar error with gcc-6.

I could make it like

    template <typename T, typename U, std::size_t N>
    constexpr map<T, U, N> make_map(std::tuple<T, U> const (&items)[N]) {
      return map<T, U, N>(bits::to_array(items));
    }

Let me know what you prefer!